bip-update-process.md

```diff
@@ -0,0 +1,516 @@
+formatting requirements specified below and named with a working title of the form "bip-authorname-topicofproposal". The
+champion must not self-assign a number to the BIP draft.
+
+BIPs that (1) adhere to the formatting requirements, (2) are on-topic, and (3) have materially progressed beyond the
+ideation phase, e.g. by generating substantial public discussion and commentary from diverse contributors, by
+independent Bitcoin projects working on adopting the proposal, or by the champion working for an extended period of time
+towards improving the proposal based on community feedback, will be assigned a number by a BIP editor.
+The BIP editors should delay number assignment when they perceive a BIP draft being met with comprehensive lack of
+interest: number assignment facilitates the distributed discussion of ideas, but before a BIP garners some interest of
+the Bitcoin community, there is no need to refer to it by a number.
+
+BIP drafts are also not ready for number assignment if they duplicate efforts, disregard formatting rules, are too
+unfocused or too broad, fail to provide proper motivation, fail to address backwards compatibility, or when the proposed
+feature’s specification is unclear or incomplete. Reviewers and BIP editors should provide guidance on how the draft may
+be improved to progress towards readiness. Pull requests that are proposing off-topic or unserious proposals, or have
+stopped to make progress may be closed.
+
+When the BIP draft is complete, a BIP editor will assign a BIP type (see below), and merge it to the BIPs repository.
+After the BIP is merged to the repository, its main focus should no longer shift significantly, even while the champion
+may continue to update the draft as necessary in the git repository. Updates to drafts by the champion should also be
+submitted as pull requests.
+
+#### Nominated[^nominated]
+
+When the champion is confident that their BIP represents a net-improvement, is
+clear, comprehensive, has a working reference implementation (if applicable),
+and is ready for adoption by the Bitcoin community, they may update the BIP’s
+status to Nominated to indicate that they recommend adoption or implementation
+of the BIP. Where applicable, the champion must ensure that any proposed
+specification is solid, not unduly complicated, and definitive. Subsequently, a
+BIP’s content should only be adjusted in minor details, e.g. to improve language, clarify ambiguities, backfill
+omissions in the specification, add test vectors for edge-cases, or address other issues discovered as the BIP is
+adopted.
+
+As time passes, substantial changes or changes that interfere with existing implementations are increasingly heavily
+discouraged.
+
+A Nominated BIP can only transition to Active or to Abandoned. If a Nominated BIP is found to need substantial
+functional changes, it should be moved to Abandoned and a new BIP should be drafted instead.
```

Comment:
The previous paragraph implied substantial changes aren't necessarily forbidden.

Reply:
Ah, I overlooked this line when I added Active back. It is still from when Nominated was the last Status BIPs would achieve. I’ll probably have to rewrite that or move that.

Reply:
I think this should be addressed